pkg/controller/util_test.go

```diff
@@ -70,9 +71,9 @@ func TestCloneController_pvcFromKey(t *testing.T) {
 
 	//create staging pvc and pods
 	pvcWithCloneRequestAnno := createPvc("target-pvc", "target-ns", map[string]string{AnnCloneRequest: "source-ns/golden-pvc"}, nil)
-	pvcUid := string(pvcWithCloneRequestAnno.GetUID())
-	sourcePod := createSourcePod(pvcWithCloneRequestAnno, DataVolName, pvcUid)
-	targetPod := createTargetPod(pvcWithCloneRequestAnno, DataVolName, pvcUid, sourcePod.Namespace)
+	id := string(pvcWithCloneRequestAnno.GetUID())
```

Comment:
not sure of the motivation to change so many occurrences of _pvcUid_ to _id_ since it is the id of the pvc? But it's a nit comment.

Reply:
Because it fails golint :)

Reply:
Really? I wonder yhy golint cares about a var named `pvcUid` vs `id`? That seems odd to me... I haven't run it in a while so I guess I need to do that today.